src/Router/AppShell.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { SFC } from "react"
+import serialize from "serialize-javascript"
+import { ContextProvider } from "../Components/Artsy"
+import { AppShellProps } from "./types"
+
+export const AppShell: SFC<AppShellProps> = props => {
+  const { loadableState, data, children, provide } = props
+
+  return (
+    <ContextProvider {...provide}>
+      <div>
+        <div
+          // FIXME: type
+          {...{ suppressHydrationWarning: true }}
```

Comment:
Is there a specific reason to spread here?

Reply:
This was a quick way to get around a missing type, need to update